src/modules/vtol_att_control/standard.cpp

```diff
@@ -368,6 +368,12 @@ void Standard::fill_actuator_outputs()
 
 	_actuators_out_1->control[actuator_controls_s::INDEX_THROTTLE] = _pusher_throttle;
 
+	/* translate MC actuators to FW actuators */
+	if (_vtol_schedule.flight_mode == TRANSITION_TO_FW || _vtol_schedule.flight_mode == TRANSITION_TO_MC) {
+		_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = -_actuators_mc_in->control[actuator_controls_s::INDEX_YAW];
+		_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] = _actuators_mc_in->control[actuator_controls_s::INDEX_THROTTLE] - 0.5f;
```

Comment:
I can see that, as with other trims, that some airframes will need a pitch trim during the transition. Let's make it a parameter, scale that with thrust, and add it to the output. The output will be controlled by TECS eventually, adding the last bit of control when active.